tests/python/pants_test/backend/graph_info/tasks/test_filter.py

```diff
@@ -304,4 +306,55 @@ def test_filter_tag_regex(self):
     self.assert_console_raises(TaskError,
       targets=self.targets('::'),
       options={'tag_regex': ['abc)']}
+                               )
+
+
+class FilterTestForTargetAlias(BaseFilterTest):
+
+  def setUp(self):
+    super(FilterTestForTargetAlias, self).setUp()
+    init_subsystem(Target.Arguments)
+
+    def add_to_build_file(type_alias, path, name, *deps):
+      all_deps = ["'{0}'".format(dep) for dep in deps]
```

Comment:
Can use an f-string!

```suggestion
      all_deps = [f"'{dep}'" for dep in deps]
```